src/main/java/net/openhft/chronicle/wire/AbstractMarshallableCfg.java

```diff
@@ -27,7 +27,7 @@ public abstract class AbstractMarshallableCfg extends SelfDescribingMarshallable
     @Override
     public void readMarshallable(@NotNull WireIn wire) throws IORuntimeException, InvalidMarshallableException {
         WireMarshaller wm = WireMarshaller.WIRE_MARSHALLER_CL.get(this.getClass());
-        wm.readMarshallable(this, wire, wm.defaultValue(), false);
+        wm.readMarshallable(this, wire, wm.defaultValue(), true);
```

Comment:
Might be worth doing a build-all branch to see what effect this has...

Reply:
already kicked off :)